### PR TITLE
Simplification due to sampler bindings starting at 0.

### PR DESCRIPTION
**Pull request by prideout**

### CODE REVIEW

File: libs/filamat/src/MaterialBuilder.cpp [hunk 1]
```diff
@@ -509,9 +509,7 @@ Package MaterialBuilder::build() noexcept {
 
         // Re-populate the set of sampler bindings for this API.
         filament::SamplerBindingMap map;
-        auto backend = static_cast<filament::backend::Backend>(params.targetApi);
-        uint8_t offset = filament::getSamplerBindingsStart(backend);
-        map.populate(offset, &info.sib, mMaterialName.c_str());
+        map.populate(&info.sib, mMaterialName.c_str());
```
Comment (bejado) [inline libs/filamat/src/MaterialBuilder.cpp]: I think we can move this outside of the loop now that this doesn't depend on the `targetApi`.

File: libs/filamat/src/MaterialBuilder.cpp [hunk 2]
```diff
@@ -701,9 +699,7 @@ const std::string MaterialBuilder::peek(filament::backend::ShaderType type,
...
```
Comment (bejado) [inline libs/filamat/src/MaterialBuilder.cpp]: This one too.

File: libs/filamat/src/MaterialBuilder.cpp (hunk 1 above)
Comment (prideout) [reply libs/filamat/src/MaterialBuilder.cpp]: Nice catch, I amended the commit.

File: libs/filamat/src/MaterialBuilder.cpp (hunk 2 above)
Comment (prideout) [reply libs/filamat/src/MaterialBuilder.cpp]: Nice catch, I amended the commit.

**Pull request merged**